src/config/navigation/config.ts

```diff
@@ -2,51 +2,62 @@ import * as Linking from 'expo-linking';
 
 import { NavigatorConfig, ScreenName } from '../../types';
 
-// import { tabNavigatorConfig } from './tabConfig';
-
-// export const navigatorConfig: NavigatorConfig = {
-//   type: 'tab',
-//   config: tabNavigatorConfig
-// };
-
-// const index = 2;
-
-// export const linkingConfig = {
-//   prefixes: [Linking.createURL('/')],
-//   config: {
-//     screens: {
-//       // For tab navigation choose the preferred tab by its position in the config array
-//       [`Stack${index}`]: {
-//         // The initialRouteName has to be the initial route of the chosen stack
-//         // (Home for drawer navigation, and whatever is specified in the tab config for tab navigation)
-//         initialRouteName: navigatorConfig.config.tabConfigs[index].stackConfig.initialRouteName,
-//         screens: {
-//           [ScreenName.EncounterUserDetail]: 'encounter',
-//           [ScreenName.Home]: '*'
-//         }
-//       }
-//     }
-//   }
-// };
-
-export const navigatorConfig: NavigatorConfig = {
-  type: 'drawer'
-};
-
-export const linkingConfig = {
-  prefixes: [Linking.createURL('/')],
-  config: {
-    screens: {
-      // For tab navigation choose the preferred tab by its position in the config array: AppStack -> `Stack${index}`
-      AppStack: {
-        // The initialRouteName has to be the initial route of the chosen stack
-        // (Home for drawer navigation, and whatever is specified in the tab config for tab navigation)
-        initialRouteName: ScreenName.Home,
-        screens: {
-          [ScreenName.EncounterUserDetail]: 'encounter',
-          [ScreenName.Home]: '*'
+import { tabNavigatorConfig } from './tabConfig';
+
+import { useSettingsContext } from '../../SettingsProvider';
```

Comment:
I recommend using Organize Imports here. The order can be like this to get rid of the error.

|before|after|
|--|--|
<img width="378" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/6c8bd02d-69e6-433a-ac76-63c1c7e79ee0"> | <img width="377" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/cec1d6be-74a2-4651-bc7f-fc6ef9d5be1c">

Reply:
Error is not there anymore since I use now the globalSettings method above

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/3bab9a27dd392bcd379a6b71b9663c1b7ca5e805